src/main/java/uk/gov/hmcts/darts/authorisation/component/impl/UserIdentityImpl.java

```diff
@@ -32,6 +38,8 @@ public String getEmailAddress() {
                 if (emailAddressObject instanceof String emailAddress && StringUtils.isNotBlank(emailAddress)) {
                     return emailAddress;
                 }
```

Comment:
Should there be an else statement to log an error unable to validate email address or something like that?

Reply:
Its already there right at the end of method.

Reply:
It throws an exception but does not log where the issue came from